### PR TITLE
framework/wifi_manager: Unify control interface name

### PR DESCRIPTION
**Pull request by SeonghoByeon**
- Change the way to specify the interface name according to the configuration set by wifi manager, not forced to set 'wl1'

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [b7142b400f06f9800d223ffd15700e4e2a62ca69] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [b7142b400f06f9800d223ffd15700e4e2a62ca69] - Code Rule Check (C++) OK.

File: external/wakaama/examples/client/connectivity_interface.c [hunk 1]
```diff
@@ -45,8 +45,8 @@ void get_interface_name(char *mac)
     strcpy(mac,"eth0");
 #elif defined(CONFIG_NET_802154)
     strcpy(mac,"wlan0");
-#elif defined(CONFIG_ARCH_BOARD_SIDK_S5JT200)
-    strcpy(mac, "wl1");
+#elif defined(CONFIG_WIFI_MANAGER)
+    strcpy(mac,CONFIG_WIFIMGR_STA_IFNAME);
 #endif
```
Comment (sangwon03) [inline external/wakaama/examples/client/connectivity_interface.c]: if CONFIT_WIFI_MANAGER is not defined, we can't use wireless lan interface. do we need to define default setting?

Comment (SeonghoByeon) [reply external/wakaama/examples/client/connectivity_interface.c]: @sangwon03 Change completed.

**Comment by seinfra** [comment]
Target : [72a82f37c7b251ef71e02e9198b9e737f1def206] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [72a82f37c7b251ef71e02e9198b9e737f1def206] - Code Rule Check (C++) OK.

**Pull request merged**